### PR TITLE
Add convertible-bond-reminder extension

### PR DESCRIPTION
**Pull request by toxic-johann**
## Description

Create a menu icon to query Chinese convertible bond data. That could remind people to buy the bonds.

## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

I'm just getting this error when opening the menubar, can you look into this?

convertible-bond-reminder 2023-08-17 at 11 41 33

At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

**Comment by toxic-johann** [comment]
Gotcha. I will check tonight

**Comment by toxic-johann** [comment]
image

**Comment by pernielsentikaer** [comment]
Still getting an error

convertible-bond-reminder 2023-08-21 at 08 55 08

**Comment by toxic-johann** [comment]
Oh. Sorry. Let me check

**Comment by toxic-johann** [comment]
Fixed.

**Comment by toxic-johann** [comment]
@pernielsentikaer It would be awesome if you had time to review it again. Thanks

File: extensions/convertible-bond-reminder/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "icon": "extension_icon.png",
+  "author": "toxic_johann",
+  "categories": [
+    "Finance"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "convertible-bond",
+      "title": "convertible bond",
```
Comment (pernielsentikaer) [inline extensions/convertible-bond-reminder/package.json:15]:
```suggestion
      "title": "Convertible Bond",
```

File: extensions/convertible-bond-reminder/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      "subtitle": "convertible bond",
```
Comment (pernielsentikaer) [inline extensions/convertible-bond-reminder/package.json:16]:
```suggestion
```

File: extensions/convertible-bond-reminder/src/convertible-bond.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      {bonds.map(({ SECUCODE, RATING }: { SECUCODE: string; RATING?: string }) => (
+        <MenuBarExtra.Item
+          title={`${SECUCODE}${RATING ? ` (${RATING})` : ""}`}
+          key={SECUCODE}
+          onAction={console.warn}
+        />
+      ))}
+      <MenuBarExtra.Item title={`Latest Refresh time：${cache.get("lastRefreshTime")}`} />
+      <MenuBarExtra.Item title="Refresh" onAction={updateBonds} />
+      <MenuBarExtra.Item title={`Last Background Sync Time：${cache.get("backgroundLaunchTime")}`} />
```
Comment (pernielsentikaer) [inline extensions/convertible-bond-reminder/src/convertible-bond.tsx:80]: This feels a bit clumpsy

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- The menubar icon is not looking good with a light background - you can see how I did in my [extension](https://github.com/raycast/extensions/blob/27c2a239b89bf6d5ad5ecec8a29ad6a1d5dc4714/extensions/tableplus/src/menubar.tsx#L21)

- What does it look like when there is data? Some more information about the extension in the README would be nice

- Uoi can remove the default Raycast icon

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by toxic-johann** [comment]
Still working on this.

**Pull request closed without merging**